server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const  routes = require('./controllers/homeRoutes');
+
+//import sequelize 
+const sequelize = require('./config/connection');
+const { all } = require('./controllers/homeRoutes');
+
+const app = express();
+const PORT = process.env.PORT || 3001;
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
```

Comment:
missing static reference to public folder

Reply:
added the express static